ceph/ceph_admin/jinja_templates/nfs.jinja

```diff
@@ -13,3 +13,6 @@ placement:
 {%+ if spec['spec'] -%}
 spec:{%+ for key, value in spec['spec'].items() -%}{{ nlt }}{{ key }}: {{ value }}{% endfor %}
 {% endif %}
+{%- if spec['extra_container_args'] -%}extra_container_args: {{ spec['extra_container_args'] }}
```

Comment:
if possible, can we add a validation?

Reply:
I'm thinking its better to take up validation as a separate task later on since it might require more time to figure out how we can validate it. Please let me know your thoughts.